modules/ROOT/pages/scalability/sharded-property-databases/overview.adoc

```diff
@@ -0,0 +1,50 @@
+:page-role: new-2025.10 enterprise-edition not-on-aura
+:description: This page describes property sharding and how it works.
+= Overview
+
+.Preview Feature
+[IMPORTANT]
+====
+The *property sharding feature is offered AS-IS* as described in your agreement with Neo4j and should only be used for internal development purposes.
+
+When this feature becomes generally available, you will need to upgrade to the latest Neo4j version (which may require downtime) to use the feature for non-development purposes.
+
+*Enabling the preview feature (internal parameter):* +
+By default, the sharded property database is disabled.
+Use the internal setting `internal.dbms.sharded_property_database.enabled=true` to enable it.
+
+During the Preview period, customers with active contracts may contact Neo4j Support through the standard support channels.
+Please note that any cases related to the Preview feature will be classified as Severity 4 by default, in accordance with the link:https://neo4j.com/terms/support-terms/[Support Terms].
+====
+
+== What is property sharding?
+
+Property sharding is about decoupling the properties associated with nodes and relationships, and storing them in separate graphs.
+The graph structure, comprising nodes and relationships, is stored in a single "graph shard".
+The properties associated with these nodes and relationships are distributed across multiple "property shards".
+This architecture enables the independent scaling of property data, allowing for the handling of larger volumes of properties without impacting the performance of the graph structure. +
+At the same time, it also allows for the optimization of storage for graph data, as the graph shards are designed to store more graph-centric information without the overhead of properties. +
+The sharded property database behaves like a standard database.
+It provides ACID guarantees for write transactions and full API support, i.e., Cypher language queries, driver's API, and Neo4j Java internal API.
+
+image::scalability/sharded-architecture.svg[title="High-level sketch of property sharding.", role="middle"]
```

Comment:
Can we use `foo-sharded` as the db name since we use that in subsequent sections?